### PR TITLE
automl: add generic batch predict sample

### PR DESCRIPTION
**Pull request by nnegrey**
Once merged, I'll update docs and remove language_batch_predict and vision_batch_predict

### CODE REVIEW

File: automl/batch_predict.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+	automl "cloud.google.com/go/automl/apiv1beta1"
+	automlpb "google.golang.org/genproto/googleapis/cloud/automl/v1beta1"
+)
+
+// batchPredict does a batch prediction.
+func batchPredict(w io.Writer, projectID string, location string, modelID string, inputURI string, outputURI string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// modelID := "ICN123456789..."
+	// inputURI := "gs://BUCKET_ID/path_to_your_input_csv_or_jsonl"
```
Comment (tbpg) [inline automl/batch_predict.go]: Delete `1` at the end?

File: automl/batch_predict.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	"context"
+	"fmt"
+	"io"
+
+	automl "cloud.google.com/go/automl/apiv1beta1"
+	automlpb "google.golang.org/genproto/googleapis/cloud/automl/v1beta1"
+)
+
+// batchPredict does a batch prediction.
+func batchPredict(w io.Writer, projectID string, location string, modelID string, inputURI string, outputURI string) error {
```
Comment (tbpg) [inline automl/batch_predict.go]: Can we add a test for this sample?

File: automl/batch_predict.go (hunk 1 above)
Comment (nnegrey) [reply automl/batch_predict.go]: `l` is intentional it's the type of file AutoML expects .jsonl from the docs.

File: automl/batch_predict.go (hunk 2 above)
Comment (nnegrey) [reply automl/batch_predict.go]: @davidcavazos , are you working on the tests for the rest of these samples?

Comment (davidcavazos) [reply automl/batch_predict.go]: I was working on them but paused them due to our discussion that we shouldn't be creating datasets and/or models on all of the testing projects.

**Pull request merged**